src/ucp/wireup/wireup_cm.c

```diff
@@ -127,7 +127,7 @@ ucp_cm_tl_bitmap_get_dev_idx(ucp_context_h context, uint64_t tl_bitmap)
     dev_index = context->tl_rscs[rsc_index].dev_index;
 
     /* check that all TL resources in the TL bitmap have the same dev_index */
-#if ENABLE_ASSERT
+#ifdef ENABLE_ASSERT
```

Comment:
can remove ifdef?

Reply:
unhandled

Reply:
I've changed to `#if` and then removed, so it is now initial code
done